src/components/wishlist/WishListInfo.jsx

```diff
@@ -0,0 +1,54 @@
+import styled from 'styled-components';
+
+function WishListInfo(categoryList) {
+  const list = categoryList.list;
+
+  const getCategoryInfo = () => {
+    return list.map(({ id, image, title }) => (
+      <li key={id}>
+        <img src={image}></img>
```

Comment:
```suggestion
        <img src={image} />
```
그냥 바로 닫아줘도 좋을 것 같아! (self-closing)